src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -121,6 +133,10 @@ public function handleException($exception, $request, $response)
      */
     protected function getRenderer($exception)
     {
+        if (!$this->renderer) {
+            $this->renderer = $this->config('exceptionRender') ?: ExceptionRenderer::class;
+        }
```

Comment:
I don't see the need to check for default / fallback class so early until it needs to be actually instantiated.